app/api/answers/submit.go

```diff
@@ -27,7 +27,8 @@ import (
 //
 //   * The user should have submission rights on `task_token.idItemLocal`.
 //
-//   * The attempt should allow submission (`allows_submissions_until` should be a time in the future).
+//   * The attempt should allow submission (`results.started_at != NULL and
```

Comment:
The `started_at` is not really about allowing submissions. One should not arrive here (having a token) without the result being started, that's all.

Reply:
fixed